bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestListTables.java

```diff
@@ -85,8 +88,34 @@ public void testTableNames() throws IOException {
         Assert.assertTrue(ArrayUtils.contains(tableList, tableName2));
       }
 
-      checkColumnFamilies(admin, tableName2);
+      {
+        TableName[] tableList = admin.listTableNames(Pattern.compile("list_table1-.*"));
+        Assert.assertTrue(ArrayUtils.contains(tableList, tableName1));
+        Assert.assertFalse(ArrayUtils.contains(tableList, tableName2));
```

Comment:
You just explained what the code is doing, I understand that.
Anyway, we can have a separate discussion about this. Let's not delay this PR because of this.